extensions/typescript-language-features/package.json

```diff
@@ -685,6 +685,12 @@
           "default": true,
           "description": "%configuration.tsserver.useSeparateSyntaxServer%",
           "scope": "window"
+        },
```

Comment:
FWIW, in VS we didn't expose this as an option - we just increased it.  It's a cap and not an amount to allocate, so there was (as far as we could tell) no cost to increasing it without using it.

Reply:
That's not entirely true, as far as I know from my experience. My understanding is that Node will only trigger a GC if it hits this limit, so a process that cycles through memory will eventually hit this limit.